StarterProject/pageobjects/common.js

```diff
@@ -0,0 +1,87 @@
+import Page from "./page.js";
+
+export default class Common extends Page {
+
+    constructor() {
+        super();    
+    }
+
+    closeWelcomeModal(){
+        let close = "div.emailPopupRefresh img";
+
+        if (this.isWelcomeModalPresent()){
+            browser.element(close).click;
+        }
+    }
+
+    closeWelcomeModalWhenPresent(){
+        let emailModal ="#monetate_allinone_lightbox";
+        browser.call(()=>{
+            browser.isVisible(emailModal).then(()=>{
+                this.closeWelcomeModal();
+            });
+        });
+    }
+
+    isWelcomeModalPresent(){
+
+        let emailModal ="#monetate_allinone_lightbox";
+
+        if (browser.isVisible(emailModal)){
+            return true;
+
+        }else{
+            return false;
+        }
+    }
+    
+    urlCurrentPage(){
+        let urlCurrentPage = browser.getUrl();  
+        return urlCurrentPage;
+    }
+
+    checkProductUrl(productid){
+        let urlCurrentPage = browser.getUrl();	
+        this.productid = productid;
```

Comment:
I get really paranoid about state... what's the life cycle of this object like?  Is there  chance this assignment could carry over between tests?